core/src/Node.ts

```diff
@@ -2,18 +2,49 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
+
+import * as fs from "fs";
+
+import {
+  Code,
+  CodeSpec,
+  ElementAspectProps,
+  ElementProps,
+  IModel,
+  InformationPartitionElementProps,
+  ModelProps,
+  RelatedElement,
+  RelatedElementProps,
+  RelationshipProps,
+  RepositoryLinkProps,
+  SubjectProps
+} from "@itwin/core-common";
+
+import {
+  ElementAspectDMO,
+  ElementDMO,
+  RelatedElementDMO,
+  RelationshipDMO
+} from "./DMO";
+
+import {
+  InformationPartitionElement,
+  Model,
+  RepositoryLink,
+  Subject,
+  SubjectOwnsPartitionElements,
+  SubjectOwnsSubjects
+} from "@itwin/core-backend";
```

Comment:
No this was me. I added `sort-imports` to the `eslint` configuration, and it's easiest to sort the imports in VS Code by selecting the lines and _> sort ascending_ at the command prompt. It's also better for diffing, because you can see exactly which imports have changed. But with the amount of imports we have in Core and our own libraries, qualified imports probably make more sense. I did that in `fir`.

Reply:
I feel like we should be consistent with the imports and stay on single line imports

Reply:
Are there other benefits of single-line imports besides consistency with iTwin libraries?

- They make diffing difficult.
- They're compact, but I'm not sure compactness is a pro.
    - Imports are meant to be read because they capture the scope of a program.
    - VS Code only updates imports correctly sometimes, especially with TypeScript's `import type`, and fiddling with lines that are hundreds of characters is a pain.

Reply:
@jackson-at-bentley I looked at the vs code source (written by typescript folks for typescript in typescript) and typescriptlang.org  I only saw examples of imports in line.  See 1 & 2 below.  Also if one of your concerns is diffing, what about the diff from inline to stacked (one per line).  That diff is difficult to follow too.  My experience w/ modern diff tools is that they generally do a pretty good job of identifying changes within a single line.

1. VS code source

https://github.com/microsoft/vscode/blob/main/src/vs/editor/test/common/view/overviewZoneManager.test.ts
import { ColorZone, OverviewRulerZone, OverviewZoneManager } from 'vs/editor/common/viewModel/overviewZoneManager';

https://github.com/microsoft/vscode/blob/main/test/smoke/src/utils.ts
import { Application, ApplicationOptions, Logger } from '../../automation';

2. TypeScript documentation

https://www.typescriptlang.org/docs/handbook/modules.html#explicitly-list-imported-names

import { SomeType, someFunc } from "./MyThings";

import { Calculator, test } from "./Calculator";

Reply:
Reverted.